pkg/kritis/util/util.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/grafeas/kritis/pkg/kritis/container"
 	"github.com/grafeas/kritis/pkg/kritis/metadata"
 	"github.com/grafeas/kritis/pkg/kritis/secrets"
+	attestationpb "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/attestation"
 	"google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/grafeas"
 	pkg "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/package"
```

Comment:
I would just call it attestation. Having pb as surffix is a more Google internal convention for protocol buffer definition.

Reply:
I wanted to name it attestation, but we are already importing a separate package called attestation. So I named it attestationpb to disambiguate